packages/pigeon/platform_tests/alternate_language_test_plugin/android/src/main/java/com/example/alternate_language_test_plugin/CoreTests.java

```diff
@@ -708,6 +708,53 @@ ArrayList<Object> toList() {
     }
   }
 
+  /**
+   * A data class containing a List, used in unit tests.
+   *
+   * <p>Generated class from Pigeon that represents data sent in messages.
```

Comment:
where is this <p> coming from?

Reply:
If we're not generating it, I guess the autoformatter?

Reply:
I just looked into it and it seems to be standard practice for doc comment paragraph separation. Might be a good idea to add it to our comment parsing tools